.github/workflows/create-dev-tag.yml

```diff
@@ -35,8 +35,14 @@ jobs:
           ref: ${{ matrix.branch }}
           depth: 0
 
-      - name: Setup asdf
-        uses: asdf-vm/actions/setup@v2
+      - name: Install NIX
+        uses: cachix/install-nix-action@v24
+
+      - name: Install devbox
+        uses: jetpack-io/devbox-install-action@v0.7.0
+        with:
+          enable-cache: true
+          skip-nix-installation: true
```

Comment:
Why do this?

Reply:
Why to skip `nix` installation? I think there is an issue with installing it on our self hosted runners. The other step `cachix/install-nix-action` works fine.